source/CoordinateConversion/ArcMapAddinCoordinateConversion/ViewModels/CollectTabViewModel.cs

```diff
@@ -318,17 +346,17 @@ private void AddFeatureLayerToMap(IFeatureClass fc)
         }
 
         private SaveFileDialog sfDlg = null;
-        private string PromptSaveFileDialog()
+        private string PromptSaveFileDialog(string ext, string filter, string title)
         {
             if (sfDlg == null)
             {
                 sfDlg = new SaveFileDialog();
                 sfDlg.AddExtension = true;
                 sfDlg.CheckPathExists = true;
-                sfDlg.DefaultExt = "kmz";
-                sfDlg.Filter = "KMZ File (*.kmz)|*.kmz";
+                sfDlg.DefaultExt = ext; // "kmz";
```

Comment:
Maybe add "Example: "kmz" so it doesn't just look like forgotten commented out code